ci/tasks/run-bats.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -e
+
+cp /usr/local/bin/bosh /usr/local/bin/bosh2
+gem install bundle
+
+echo "Setting up artifacts..."
+cp ./stemcell/*.tgz stemcell.tgz
+
+# path to the stemcell you want to use for testing
+export BAT_STEMCELL="${PWD}/stemcell.tgz"
+
+# path to the bat yaml file which is used to generate the deployment manifest
+export BAT_DEPLOYMENT_SPEC="${PWD}/oci-config/bat.yml"
+
+# BOSH CLI executable path
+export BAT_BOSH_CLI=`which bosh2`
+
+export BAT_DNS_HOST=8.8.8.8
+
+# the name of infrastructure that is used by bosh deployment. Examples: aws, vsphere, openstack, warden.
+export BAT_INFRASTRUCTURE=oci
+
+# the type of networking being used: dynamic or manual.
+export BAT_NETWORKING="manual"
+
+# the path to ssh key, used by OS specs to ssh into BOSH VMs
+export BAT_PRIVATE_KEY="${PWD}/oci-config/cpikeys/id_rsa"
```

Comment:
We should use the userkey for this and not the cpi key.

Reply:
ok